app/Http/Regions/DestinationHeader.php

```diff
@@ -28,10 +28,16 @@ public function render($destination)
                     ->is('large')
                     ->with('title', $destination->name)
                 )
+                ->pushWhen($user && $user->hasRole('admin'),
+                    component('MetaLink')
+                        ->is('white')
+                        ->with('title', trans('content.action.edit.title'))
+                        ->with('route', route('destination.edit', [$destination]))
+                )
                 ->push(component('Body')
                     ->is('white')
                     ->is('responsive')
-                    ->with('body', $destination->vars()->description)
+                    ->with('body', $destination->description ? $destination->description : $destination->vars()->description)
```

Comment:
Also, extract the whole logic to a variable and also add add user name, something like

description ends here ([username])